src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { PageNumber, PaginationWrapper } from './style';
+
+function Pagination({ onClickPage, totalPages, page }) {
```

Comment:
Change to PureComponent.
Make it configurable, like if I want to display 5 numbers at the end and 2 at the start...
Please, refactor it: self-executed function is a bloody hell :)